arch/x86_64/mcount.S

```diff
@@ -1,8 +1,54 @@
 /* argument passing: %rdi, %rsi, %rdx, %rcx, %r8, %r9 */
 /* return value: %rax */
 /* callee saved: %rbx, %rbp, %rsp, %r12-r15 */
-/* stack frame (with -pg): return addr = 8(%rbp), prev fp = 0(%rbp) */
-/* stack frame (with -fentry): return addr = (%rsp), prev fp = %rbp */
+/* stack frame (with -pg): parent addr = 8(%rbp), child addr = 56(%rsp) */
```

Comment:
@namhyung ,
If there isn't the problem about the 16-byte stack alignment,
we could just use `push` and `pop` insn.
However, because it is required from GCC7, we should manually adjust stack pointer `%rsp`, right ?